xwiki-platform-core/xwiki-platform-user/xwiki-platform-user-script/src/main/java/org/xwiki/user/script/GroupScriptService.java

```diff
@@ -147,16 +147,16 @@
     }
 
     /**
-     * Checks if a group (candidate) can be added to the members of another group (target).<br />
+     * Checks if a group (candidate) can be added to the members of another group (target).<br>
      * A group (candidate) can be added to the members of another group (target) if the target is not, transitively,
-     * already part of the members of candidate.<br />
-     * A group cannot be added to its own members.<br />
-     * A group (candidate) cannot be added to the members of a group (target) if candidate is already one of the direct
-     * members of target.<br />
+     * already part of the members of the candidate.<br>
```

Comment:
why don't you just remove those br tags? they are not really useful are they?

Reply:
if you need a list you can use `<ul>` and `<li>`

Reply:
In general think of Javadoc as a piece of HTML with a few helpers based on the `{@something}` syntax.

Reply:
It's not HTML. You don't close elements for example and only a small subset is supported.

BR should never be used. If you need items you should use a list.

Reply:
Javadoc updated with ul/li tags. For now on I'll add a `mvn javadoc:javadoc` to my pre-push tests.

Reply:
> BR should never be used. If you need items you should use a list.

My point is that to would not do a list using `*` and `<br/>` in HTML.